.travis/deploy.py

```diff
@@ -13,7 +13,7 @@
 
 
 HOME = expanduser("~")
-DEFAULT_SERVER = 'https://pypi.python.org/pypi'
+DEFAULT_SERVER = 'https://upload.pypi.org/legacy/'
```

Comment:
I don't know what's the background of this change, but switching to something with `legacy` in URL looks slightly alarming to me.

Reply:
it's counter-intuitive but okay!
```(legacy in this URL refers to the fact that this is the new server implementation’s emulation of the legacy server implementation’s upload API.)```
https://packaging.python.org/guides/migrating-to-pypi-org/

Reply:
Nudge @mhadam , then ask Anton to re-review

Reply:
To fix this, I removed the DEFAULT_SERVER variable in `.travis/deploy.sh`.